examples/gallery/images/grdgradient.py

```diff
@@ -0,0 +1,59 @@
+"""
+Calculating the gradient of a grid file
+--------------------
```

Comment:
```suggestion
Calculating the gradient and radiance of a grid file
----------------------------------------------------
```
The underlines need to be at least as long as the title, otherwise sphinx will throw an error.

I would recommend adjusting the title to specify that the example is not only calculating the gradient.